util/promote_staging_data

```diff
@@ -452,27 +464,36 @@ log "Data integrity test results for all teams:"
 echo -e "team\tall_tests_passed"
 echo -e "${all_teams_results}"
 
-# Try syncing staging data to production
 while read -r team result_status; do
   if [[ "$result_status" == True ]]; then
     staging_bucket=gs://asap-${STAGING_BUCKET_TYPE}-data-${team}
     production_bucket=gs://asap-curated-data-${team}
```

Comment:
Will we ever go DEV -> UAT?

Reply:
@hkeward This was part of the original script you wrote. AFAIK we haven't encountered this yet.